### PR TITLE
[WIP]New package: helvum-0.3.1

### PR DESCRIPTION
**Pull request by xaltsc**
This package does not build because rust is too old. Waitin for #32555 to be merged.

#### Testing the changes
- I tested the changes in this PR: **YES**|**briefly**|**NO**

### CODE REVIEW

**Comment by Roger-Roger-debug** [comment]
Shouldn't the homepage be [https://gitlab.freedesktop.org/ryuukyu/helvum](https://gitlab.freedesktop.org/ryuukyu/helvum)?

**Comment by dkwo** [comment]
Would you like to try again now?

**Review by classabbyamp — CHANGES_REQUESTED** [review]
I was able to successfully build and install `helvum` on x86_64 glibc. However, upon launching, I get this error and nothing else happens:

```
$ helvum 
helvum: symbol lookup error: /usr/lib64/libgtk-4.so.1: undefined symbol: pango_attr_word_new
```

Also, v0.3.2 is available from upstream now, and it has the same error as above

**Comment by Roger-Roger-debug** [comment]
`libgtk-4.so.1` is provided by gtk4, so it should be added as a dependency (maybe even pipewire?). Also when compiling I get the warning
```
warning: the cargo feature `edition2021` has been stabilized in the 1.56 release and is no longer necessary to be listed in the manifest
```
so we can probably remove the configure bit.

**Comment by dkwo** [comment]
This builds and installs fine for me on `x86_64-musl`.
```
# Template file for 'helvum'
pkgname=helvum
version=0.3.4
revision=1
build_style="meson"
hostmakedepends="cargo pkg-config"
makedepends="clang gtk4-devel pipewire-devel desktop-file-utils
 glib-devel appstream-glib-devel"
depends="gtk4 pipewire dbus"
short_desc="GTK patchbay for pipewire"
maintainer="xaltsc <hey+dev@xaltsc.dev>"
license="GPL-3.0-only"
homepage="https://gitlab.freedesktop.org/ryuukyu/helvum"
distfiles="https://gitlab.freedesktop.org/ryuukyu/helvum/-/archive/${version}/helvum-${version}.tar.gz"
checksum=e19054f3bcd1ba234c451134fe7f4c6afee35f108c122642c1d502fc16785c02
```
Anyone @xaltsc @Roger-Roger-debug @classabbyamp interested in updating this PR, or creating a new one?

**Comment by dkwo** [comment]
Well, pipewire requires dbus anyway, right?

**Comment by xaltsc** [comment]
Sorry for the delay, given that this PR #35591 seems to be going well, should I close this PR or can I help in some way ?

**Comment by classabbyamp** [comment]
I think that PR is just waiting for a maintainer approval. If you'd like to test it that'd be great but otherwise I don't think there's anything to do.

**Pull request closed without merging**
